pick.1

```diff
@@ -8,6 +8,7 @@
 .Nm
 .Op Fl hvKS
 .Op Fl d Op Fl o
+.Op Fl V
```

Comment:
Uppercase letters are by convention used to disable a feature in pick. I guess you chosed `V` since `v` already is taken. Enabling Vi-mode might therefore be done using a separate key binding.

Reply:
Didn't know about this convention and you guessed right ... ;-) I'm thinking of `-y` as the command line switch to enable the `vi`-mode at start ... a separate key binding would maybe inconvenient when `pick` is called from a script (`pick.vim`) ...